packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -107,6 +108,14 @@ export interface StashClient {
 	clientId: string;
 }
 
+/**
+ * @internal
+ */
+export interface HandleCreated {
+	type: "handleCreated";
+	handles: IFluidHandle[];
```

Comment:
why an array? i think one would be enough

Reply:
i could see the generator having and array of handles, and randomly picking on to hand out